File: taco-ui/src/app/model/ingredient.ts
export class Ingredient {
  id: String
  name: String;
  type: String;
  price: String;


  constructor(id: String, name: String, type: String, price: String) {
    this.id = id;
    this.name = name;
    this.type = type;
    this.price = price;
  }
}

File: taco-email/src/main/java/com/example/mail/model/Ingredient.java
package com.example.mail.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor(force = true)
public class Ingredient {
    private final String id;
    private final String name;
}
